reinforcement_learning/rl_game_server_autopilot/game-env/minecraft-server-image/start.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+import sys
+import os
+import socket
+import random
+import subprocess
+import boto3
+import json
+import signal
+from ec2_metadata import ec2_metadata
+
+# global variables 
+region='us-west-2'
+public_hostname=''
+public_port=''
+
+# Get the service resource
+'''
+sqs_cli=boto3.resource('sqs',region_name='us-east-1')
```

Comment:
Should this be "us-east-1"? I think everywhere we are dealing with region, we should either read it from environment_variables, or passing as a command line argument.